website/search/elastic_search.py

```diff
@@ -85,6 +85,9 @@ def wrapped(*args, **kwargs):
                 if 'ParseException' in e.error:
                     raise exceptions.MalformedQueryError(e.error)
                 raise exceptions.SearchException(e.error)
+            except TransportError as e:
```

Comment:
Since `RequestError` is a [subclass](https://github.com/elastic/elasticsearch-py/blob/1.3.0/elasticsearch/exceptions.py#L85) of `TransportError`, we could broaden the previous `except` and drop this one (the risk of matching a non-`RequestError` `ParseException` seems slight).